README.rst

```diff
@@ -44,3 +44,48 @@ or, alternatively, run it directly against the url ::
 
 The project then should have access to the ``tequila-dokku`` role in
 its playbooks.
+
+Variables
+---------
+
+The following variables can be used to control behavior. They are
+listed with their original defaults.
+
+Debconf options (see `here <http://dokku.viewdocs.io/dokku/getting-started/install/debian/#debconf-options>`_ for docs)::
+
+    dokku_vhost_enable: "true"
+    dokku_web_config: "false"
+    dokku_skip_key_file: "true"
+    dokku_key_file: ""
+    dokku_hostname: 'dokku.me'
+
+Docker and dokku versions, and whether to do
+an upgrade (otherwise does an install)::
```

Comment:
I see the docker and dokku versions, but not anything that specifies whether to do an upgrade.